stl/inc/yvals_core.h

```diff
@@ -163,10 +163,12 @@
 // P1227R2 Signed std::ssize(), Unsigned span::size()
 //     (partially implemented)
 // P1357R1 is_bounded_array, is_unbounded_array
+// P1456R1 Move-only Views
 // P1612R1 Relocating endian To <bit>
 // P1651R0 bind_front() Should Not Unwrap reference_wrapper
 // P1690R1 Refining Heterogeneous Lookup For Unordered Containers
 // P1754R1 Rename Concepts To standard_case
+// P1870R1 forwarding-range Is Too Subtle

```

Comment:
See #39, this should be listed as `P1870R1 safe_range`. The paper's original name described the problem that it solved; we prefer to list papers according to what they add.